packages/blocks/src/text-input/validations.js

```diff
@@ -8,4 +8,14 @@ export default {
 			return 'Please enter a valid email address.';
 		}
 	},
+
+	urlValidation: ( value ) => {
+		if (
+			! value.match(
+				/[(http(s)?):\/\/(www\.)?a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/gi
+			)
+		) {
+			return 'Please enter a valid url.';
```

Comment:
Needs i18n.

I'd also repeat the question regarding the use of all caps on acronyms, should it be "URL"?